release/manifest.json

```diff
@@ -12,5 +14,10 @@
     "default_popup": "popup.html"
   },
   "background": {},
-  "permissions": ["https://www.pixiv.net/"]
+  "permissions": [
+    "webRequest",
+    "webRequestBlocking",
+    "https://www.pixiv.net/",
+    "https://i.pximg.net/"
```

Comment:
Sorry, this line is very important but was omitted at the first time.

Reply:
I will merge this pr.